.golangci.yml

```diff
@@ -145,16 +151,16 @@ linters-settings:
     # Enable to require an explanation of nonzero length after each nolint directive. Default is false.
     require-explanation: true
     # Enable to require nolint directives to mention the specific linter being suppressed. Default is false.
-    require-specific: true
+    require-specific: false
```

Comment:
At least one line was failing in 3+ linters, but they're reported one at a time... It looks too strict to a repo like Beats.